Sources/SwiftKuery/Table.swift

```diff
@@ -159,16 +288,50 @@ open class Table: Buildable {
         return self
     }
     
-    /// Add a primary key to the table.
-    ///
-    /// - Parameter columns: Columns that constitute the primary key.
+    /**
+     Function to set a single `Column` instances` as a primary key, in the `Table` instance.
+     This function calls the composite primaryKey function with a single column to create a single primary key.
+     ### Usage Example: ###
+     In this example, a column for id is initialized and a `Table` instance called "personTable" is created.
```

Comment:
We can cut this text down - how about:
"In this example, the primary key is set to the `id` column for the table `personTable`."